### PR TITLE
Views/aj 221 get team views

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

File: src/main/java/clickup/requests/ViewsRequest.java [hunk 1]
```diff
@@ -14,8 +14,12 @@
 import clickup.entities.features.views.View;
 import clickup.entities.features.views.Views;
 import com.fasterxml.jackson.core.JsonProcessingException;
+import core.api.ApiManager;
+import core.api.ApiMethod;
 import core.api.ApiRequest;
```
Comment (GustavoZHA) [inline src/main/java/clickup/requests/ViewsRequest.java]: Delete blank line

File: src/test/java/cucumber/steps/ViewsSteps.java [hunk 2]
```diff
@@ -10,29 +10,45 @@
...
 import core.api.ApiMethod;
 import core.api.ApiRequest;
 import core.api.ApiRequestBuilder;
 import core.api.ApiResponse;
-import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.json.JSONObject;
+import org.testng.asserts.SoftAssert;
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/ViewsSteps.java]: Delete here too

File: src/main/java/clickup/requests/ViewsRequest.java (hunk 1 above)
Comment (Raymundogs) [reply src/main/java/clickup/requests/ViewsRequest.java]: Solved, thanks.

File: src/test/java/cucumber/steps/ViewsSteps.java (hunk 2 above)
Comment (Raymundogs) [reply src/test/java/cucumber/steps/ViewsSteps.java]: Solved it. Thanks.

**Review by JorgeRCV — APPROVED** [review]

File: src/test/java/cucumber/steps/ViewsSteps.java [hunk 3]
```diff
@@ -10,29 +10,45 @@
...
     private ApiResponse apiResponse;
     private ApiRequest apiRequest;
     private ScenarioTrash scenarioTrash;
+    private int actual;
+    private ApiResponse response = new ApiResponse();
+    private ScenarioContext scenarioContext = ScenarioContext.getInstance();
+    private SoftAssert softAssert;
 
-    public ViewsSteps(ApiRequestBuilder apiRequestBuilder, ScenarioTrash scenarioTrash, ApiResponse apiResponse) {
+    public ViewsSteps(ApiRequestBuilder apiRequestBuilder, ScenarioTrash scenarioTrash, ApiResponse apiResponse,
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/ViewsSteps.java]: Is nesesary to use pico container to ScenarioTrash? ScenarioTrash is singleton

File: src/test/resources/features/views/get_all_views.feature [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Feature: Get all Views
+
+  @DeleteViews
+  Scenario: As a user I want to get all Team Views and verity the quantity
+    Given I get the initial amount of team views
+    And I set the views with request endpoint to /team/{team_id}/view
```
Comment (GustavoZHA) [inline src/test/resources/features/views/get_all_views.feature]: Add space in the "And" according the convencions

File: src/test/resources/features/views/get_all_views.feature [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    When I add the amount of 2 to the total of team views
+    And I execute the GET request for views
+    Then I verify the amount of views has increased by 2
+    And I verify that the response status is 200
```
Comment (GustavoZHA) [inline src/test/resources/features/views/get_all_views.feature]: Add new line in the file

File: src/test/java/cucumber/steps/ViewsSteps.java (hunk 3 above)
Comment (Raymundogs) [reply src/test/java/cucumber/steps/ViewsSteps.java]: You are right, I'll fix it.

File: src/test/resources/features/views/get_all_views.feature (hunk 5 above)
Comment (Raymundogs) [reply src/test/resources/features/views/get_all_views.feature]: I'll add it, thanks.

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**